portal-web/test/functional/com/liferay/portalweb/macros/ABTest.macro

```diff
@@ -81,29 +110,32 @@ definition {
 	}
 
 	macro deleteABTest {
-		Click(locator1 = "ABTest#KEBAB_AB_TEST");
-
-		MenuItem.clickNoError(menuItem = "Delete");
-
-		AssertTextEquals.assertPartialText(
-			locator1 = "Modal#BODY",
-			value1 = "Are you sure you want to delete this? It will be deleted immediately.");
+		if (IsElementNotPresent(locator1 = "Modal#MODAL_CONTENT")) {
+			if (IsElementPresent(locator1 = "ABTest#KEBAB_AB_TEST")) {
+				Click(locator1 = "ABTest#KEBAB_AB_TEST");
+
+				MenuItem.clickNoError(menuItem = "Delete");
+			}
+			else {
+				Click(locator1 = "Button#TRASH_ENABLED");
+			}
+		}
 
-		Click(
-			key_text = "Delete",
-			locator1 = "Modal#MODAL_FOOTER_BUTTON");
+		Click(locator1 = "ACSettings#CONFIRMATION_BUTTON");
 
-		Alert.viewSuccessMessage();
+		Alert.viewSuccessMessageText(successMessage = "Success:Your request completed successfully.");
 	}
 
-	macro deleteABTestInAC {
-		Click(
-			key_text = "ellipsis-v",
-			locator1 = "Icon#ANY");
+	macro deleteABTestACSide {
+		Button.clickDelete();
 
-		MenuItem.clickNoError(menuItem = "Delete Test");
+		AssertTextEquals.assertPartialText(
+			locator1 = "Modal#MODAL_CONTENT",
+			value1 = "Are you sure you want to delete this test? You will permanently lose all test data and results collected from this test. You will not be able to run this test. You will not be able to undo this operation.");
 
-		Button.clickDelete();
+		Click(locator1 = "ACSettings#CONFIRMATION_BUTTON");
+
+		Alert.viewSuccessMessageText(successMessage = "Success:The test has been deleted.");
```

Comment:
Hey @manuferreira, I think you moved this macro to `ACTests.macro` and forgot to remove it from this other file

Commit: https://github.com/liferay-ac/liferay-portal/pull/729/commits/c02f892fdc508bba791eb310c3a23ebdf2777612

Reply:
oh god, I totally forgot, thanks!

Reply:
https://github.com/liferay-ac/liferay-portal/pull/729/commits/c68c369a1b4fad48d8b5cd6d3a43d09c986471bb